_layouts/main.html

```diff
@@ -58,28 +58,30 @@ <h2>Meetings</h2>
 
 
  <div class="col-lg-4">
-  <h2>HSF at CHEP</h2>
+  <h2>New Working Groups</h2>
     <p>
-    The HSF had a great <a href="http://chep2018.org/">CHEP 2018</a>. We organised the highly
-    successful <a href="https://indico.cern.ch/event/694818/">PyHEP Workshop</a> in advance of 
-    CHEP.<br>
-    &nbsp;&nbsp; At CHEP itself the HSF Comminity White Paper was mentioned in many, many talks and Michel Jouvin
-    gave a great <a href="https://indico.cern.ch/event/587955/contributions/3012294/">plenary talk</a> 
-    that went beyond the CWP to describe the next steps for HEP Software and Computing.<br>
-	&nbsp;&nbsp; Important common software projects featured highly as well, with a 
-	<a href="https://indico.cern.ch/event/587955/contributions/3012710/">fine talk on common
-	tracking software</a> from Andi Salzburger.
+    The HSF is establishing new working groups with a
+    <a href="/organization/working-group-mandates.html">mandate</a>
+    to pursue the goals from the
+    <a href="/organization/cwp.html">Community White Paper Roadmap</a>
+    last year. We are
```

Comment:
Tiny/picky suggestion: groups with <...>mandates</a> to … Roadmap</a> published last year on the <link to arxiv>arXiv</a>.